bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -15,8 +15,6 @@
 //! ## Consensus Updates
 //!
 //! * [`Call::submit`]: Submit a finalized beacon header with an optional sync committee update
-//! * [`Call::submit_execution_header`]: Submit an execution header together with an ancestry proof
```

Comment:
This does not seem relevant?

Reply:
Correct, reverted in https://github.com/paritytech/polkadot-sdk/pull/3727/commits/27ecafd7698295cf6b8e0f67a556bc1f9a23afbb.